docling_core/transforms/visualizer/layout_visualizer.py

```diff
@@ -0,0 +1,195 @@
+"""Define classes for layout visualization."""
+
+from copy import deepcopy
+from typing import Optional, Union
+
+from PIL import ImageDraw, ImageFont
+from PIL.Image import Image
+from PIL.ImageFont import FreeTypeFont
+from pydantic import BaseModel
+from typing_extensions import override
+
+from docling_core.transforms.visualizer.base import BaseVisualizer
+from docling_core.types.doc import DocItemLabel
+from docling_core.types.doc.base import BoundingBox
+from docling_core.types.doc.document import ContentLayer, DocItem, DoclingDocument
+from docling_core.types.doc.page import TextCell
+
+
+class Cluster(BaseModel):
+    """Cluster."""
+
+    id: int
+    label: DocItemLabel
+    bbox: BoundingBox
```

Comment:
Let's make this a BoundingRectangle. Every BoundingRectangle can be mapped into a BoundingBox. If we ever want rotated BoundingBox, we at least have the right type from the start.

Reply:
Done.